src/loginServices.js

```diff
@@ -748,9 +748,14 @@ export function requestLogin(okapiUrl, store, tenant, data) {
  * @returns {Promise} Promise resolving to the response of the request
  */
 function fetchUserWithPerms(okapiUrl, tenant, token) {
+  const rtrIgnore = !token;
+
   return fetch(
     `${okapiUrl}/bl-users/_self?expandPermissions=true&fullPermissions=true`,
-    { headers: getHeaders(tenant, token) },
+    {
+      headers: getHeaders(tenant, token),
+      rtrIgnore,
```

Comment:
This would build in an rtrIgnore flag that could be added to any fetch call... this doesn't seem like the right approach.

Could `/bl-users/_self` just be added to the `permissible` list?

Reply:
I don't think we can as we need to refresh tokens when users restore page, but for SSO is exception because we get session right after login

Reply:
> that could be added to any fetch call

that's true, but as I mentioned in another comment, we need to create temporary okapiSession

but in general `rtrIgnore` just simplifies skipping RTR if really required as in any place it's possible to change token expiration values, why can it be done? I have no idea )